Topics/Tech_Stacks/Postman_Backend_Testing.md

```diff
@@ -0,0 +1,64 @@
+
+# Postman Backend Testing
+
+In this Postman application overview, we will go over how to 
+install the application and the details of how to tests your API endpoints.
+
+## Download and installation
+the following link will take you to the postman website where you can
+download the application if you do not have it already installed:
+[Download](https://www.postman.com/downloads/)
+
+
+
+After the download the following video will help you install the application:
+
+[Windows](https://www.youtube.com/watch?v=pBo_oClYjjM&ab_channel=GeekyScript)  
+[Mac](https://www.youtube.com/watch?v=sriayOzVaMM&ab_channel=Postman)  
+[Linux](https://www.youtube.com/watch?v=9ZZdl4CQbCg&ab_channel=OSTechHelp)
+
+
+## Testing with Postman
+### Creating a workspace
+Before starting to test your API, it is best to create a workspace for the 
+specific project that you are working on. This way team members can access
+and contribute to the test cases.
+To do this, click on the workspace tab and then click on the 
+'Create Workspace' button and name your workspace.
+then click on the newly created workspace to write your requests within it.
+### Creating a Collection
+To send request within this workspace. You need to create a new
+collection which will contain all the different requests to your API endpoint.
+Create a new Collection by clicking on the '+' that is location next to the 
+collection side tab.
+### Sending requests
+Once you have created the Collection, you can now create your request. 
+Do this by expanding the newly created collection and clicking on add request

```

Comment:
This might lead to confusion because there are two places that are labelled"New collection". To avoid this, it would be helpful if you could provide clear instructions on where the reader should expand the collection. For example, you could specify the exact location of the button